### PR TITLE
Wizard improvements

### PR DESCRIPTION
**Pull request by DavDan1**
PT: https://www.pivotaltracker.com/story/show/178520033
____
### User Story
```
As a user
In order to enjoy the visit to wizard
We improved it and made it more user friendly
```
___
### Changes proposed
___
- Also add in BOTH buttons
- remove company
- Add phone number
- Loading to submit
- More info about agent
- link to LinkedIn
- Make desktop wizard larger
____
### Movie

https://user-images.githubusercontent.com/57390886/122121300-be816a00-ce2b-11eb-850d-048d223b6efa.mp4

### CODE REVIEW

**Review by arunbhalli — APPROVED** [review]

File: src/components/Header.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ const LandingHeader = ({ landing }) => {
         <LanguageSelect />
       ) : (
         <div className='menu-wrapper'>
-          <CustomButton dataCy='phone-number'>
+          <CustomButton dataCy='phone-question'>
             <a href='tel:+46812345678'>08-123 456 78</a>
```
Comment (christerforsgren91) [inline src/components/Header.jsx]: 031 insted of 08 :)

File: src/components/Inquiry/Answer.jsx [hunk 2]
```diff
@@ -41,6 +42,7 @@ const Answer = ({
 
   const scroll = () => {
     let container = document.getElementById('wizard-container')
+    // !first &&
```
Comment (christerforsgren91) [inline src/components/Inquiry/Answer.jsx]: waz is daz?

**Review by christerforsgren91 — APPROVED** [review]

File: src/components/Inquiry/Answer.jsx (hunk 2 above)
Comment (DavDan1) [reply src/components/Inquiry/Answer.jsx]: gone

File: src/components/Header.jsx (hunk 1 above)
Comment (DavDan1) [reply src/components/Header.jsx]: done

File: src/components/Inquiry/Question.jsx [hunk 3]
```diff
@@ -3,10 +3,12 @@ import agent from '../../assets/Jehn.png'
...
-    <div className='question-container'>
-      <img src={agent} alt='agent'/>
-      <p data-cy={dataCy}>{text}</p>
-    </div>
+    <>
+      <div className='question-container'>
+        <img src={agent} alt='agent' />
+        <p data-cy={dataCy}>{text}</p>
+      </div>
+    </>
```
Comment (1CIM) [inline src/components/Inquiry/Question.jsx]: are the fragments needed?

File: src/components/Inquiry/Wizard.jsx [hunk 4]
```diff
@@ -1,41 +1,48 @@
...
 
   return (
-    <div className='wizard-container' id='wizard-container' data-cy='wizard'>
-      <InquiryCompanySize />
+    <div
+      className='wizard-container'
+      id='wizard-container'
+      data-cy='wizard'
+      tabindex='0'>
+      {<InquiryCompanySize />}
```
Comment (1CIM) [inline src/components/Inquiry/Wizard.jsx]: no needs for curlies right

File: src/components/Inquiry/Wizard.jsx [hunk 5]
```diff
@@ -1,41 +1,48 @@
...
-      {formData.company && <InquiryPeers />}
+      {formData.email && <InquiryPeers />}
       {formData.peers && <InquiryLocation />}
       {formData.locations[0] && <InquiryWorkingHours />}
+      {formData.flexible && <InquiryPhoneNumber />}
+
       {submitMessage ? (
         <>
           <Question dataCy='on-submit-message' text={submitMessage} />
         </>
```
Comment (1CIM) [inline src/components/Inquiry/Wizard.jsx]: can ommit these frags aswell right

File: src/modules/Inquiries.js [hunk 6]
```diff
@@ -2,7 +2,11 @@ import axios from 'axios'
 import store from '../state/store/configureStore'
 
 const Inquiries = {
-  async create(formData) {
+  async create(formData, setLoading) {
+    setLoading(true)
+    setTimeout(() => {
+      return
+    }, 1000000000)
```
Comment (1CIM) [inline src/modules/Inquiries.js]: is it long enough? X)

File: src/components/Inquiry/Question.jsx (hunk 3 above)
Comment (1CIM) [reply src/components/Inquiry/Question.jsx]: removed

File: src/components/Inquiry/Wizard.jsx (hunk 4 above)
Comment (1CIM) [reply src/components/Inquiry/Wizard.jsx]: removed

File: src/components/Inquiry/Wizard.jsx (hunk 5 above)
Comment (1CIM) [reply src/components/Inquiry/Wizard.jsx]: removed

File: src/modules/Inquiries.js (hunk 6 above)
Comment (1CIM) [reply src/modules/Inquiries.js]: set to 500ms

**Pull request merged**